common/lib/dependabot/pull_request_creator/message_builder/metadata_presenter.rb

```diff
@@ -244,10 +244,14 @@ def source_provider_supports_html?
           !%w(azure bitbucket codecommit).include?(source.provider)
         end
 
+        def sanitize_to_html?
```

Comment:
I added a new method here for the sanitation, but perhaps using the `source_provider_supports_html` method is better.